flake.nix

```diff
@@ -18,6 +18,11 @@
                 shell = { inherit crossPlatforms; };
               });
           })
+
+          haskellNix.overlay
+          (final: prev: prev.lib.optionalAttrs prev.stdenv.hostPlatform.isMusl {
+            libevdev = prev.libevdev.overrideAttrs (_: { dontDisableStatic = true; });
+          })
```

Comment:
I should upstream this patch somehow, as [initially suggested on Reddit](https://www.reddit.com/r/haskell/comments/wd7oqx/comment/ilds327/?context=3).